juju/juju
# juju-updateseries-refactoring

commit b9e69d1f7fc15be96704e5fc4937455f7b57901c
Author: vinodhini balusamy <me.vinob@gmail.com>
Date:   Wed May 2 11:15:10 2018 +1000

    "Corrected function named return values"

commit a2343fe354d365264b2e50032eedf654d59f0e72
Author: vinodhini balusamy <me.vinob@gmail.com>
Date:   Wed May 2 11:00:26 2018 +1000

    "Added unit test toverify systemd failure"

commit 6ed50ba7c668d483f60431a60aed3965cef277f4
Author: vinodhini balusamy <me.vinob@gmail.com>
Date:   Tue May 1 17:12:50 2018 +1000

    "Addressed review comments"

commit fdd9b970b2ed651245f828fc5a5a32ab76d53103
Author: vinodhini balusamy <me.vinob@gmail.com>
Date:   Tue May 1 15:09:26 2018 +1000

    "Fixed update-series unit tests"

commit beb765f95df98072a72495525548ee22f93f567a
Author: vinodhini balusamy <me.vinob@gmail.com>
Date:   Tue May 1 07:00:24 2018 +1000

    "Code changes to fix updateseries unit tests"

commit 4c676260dbf429815c7782b6b959f5569aedc684
Author: vinodhini balusamy <me.vinob@gmail.com>
Date:   Mon Apr 30 09:36:40 2018 +1000

    "Added call to verify symlink in unittest"

commit 2705bcb50911e052ec4fb6036e6cb2bc33d43ddc
Author: vinodhini balusamy <me.vinob@gmail.com>
Date:   Mon Apr 30 07:42:26 2018 +1000

    "Added unit tests for methods in agentconf.go"

commit 1b93841acc6c54388c5883d579faca0a3e8598ae
Author: vinodhini balusamy <me.vinob@gmail.com>
Date:   Fri Apr 27 15:08:24 2018 +1000

    "Addressed Review comments"

commit 9063b4366d78f067d44d9a0c5554afe1a234f99f
Author: vinodhini balusamy <me.vinob@gmail.com>
Date:   Fri Apr 27 12:53:42 2018 +1000

    "Addressed fmt issues"

commit 369cdfe65bba105af2467620dacd2a87c7f2c2e4
Author: vinodhini balusamy <me.vinob@gmail.com>
Date:   Fri Apr 27 12:48:19 2018 +1000

    "Replace cmd.Context to return values in ServiceFd"

commit 02eb4c656bff901a76c4087774d883dfd1cd7dc1
Author: vinodhini balusamy <me.vinob@gmail.com>
Date:   Thu Apr 26 21:47:43 2018 +1000

    "addressed review comments"

commit 2e9b38c71adbebb5d641251e36a2643826761880
Author: vinodhini balusamy <me.vinob@gmail.com>
Date:   Thu Apr 26 16:48:51 2018 +1000

    "Unit test fix for updateseries"

commit 60c3187ed5386bc64541bb0ad85a3628a41e26a7
Author: vinodhini balusamy <me.vinob@gmail.com>
Date:   Thu Apr 26 15:49:01 2018 +1000

    "Fixes for unit tests"

commit 021dc42d78f7b1e1e34faf76912b015adddab09b
Author: vinodhini balusamy <me.vinob@gmail.com>
Date:   Thu Apr 26 14:20:53 2018 +1000

    "Addressed review comments"

commit 5ec52f82c6a3a3c4f7dfa9172e1ef789512f5eb7
Author: vinodhini balusamy <me.vinob@gmail.com>
Date:   Thu Apr 26 12:13:47 2018 +1000

    "Addressed review comments"

commit bb177467019567c691687e69e2f519d8b3cbeb0c
Author: vinodhini balusamy <me.vinob@gmail.com>
Date:   Thu Apr 26 11:20:20 2018 +1000

    "updated the export function name and format issue"

commit 028ba1010e407efde3c3bd4d61d7418c8ec71826
Author: vinodhini balusamy <me.vinob@gmail.com>
Date:   Thu Apr 26 11:03:17 2018 +1000

    "Fixed the panic issue/addressed review comments"

commit 44e371c51bc1eff0a55574bc08cae6dc75593246
Author: vinodhini balusamy <me.vinob@gmail.com>
Date:   Thu Apr 26 06:51:20 2018 +1000

    "cleanedup commented code and fixed format issue"

commit db5decf62b71def6825ccfd4546e4de56e04b425
Author: vinodhini balusamy <me.vinob@gmail.com>
Date:   Thu Apr 26 06:33:22 2018 +1000

    "Refactored the code based on review discussions"

commit b0aa1f934c754f2e27fdbf5694ec1e2d32e16dd6
Author: vinodhini balusamy <me.vinob@gmail.com>
Date:   Tue Apr 24 10:43:59 2018 +1000

    "Addressed the fromat issue"

commit 211487137d052ccdd98f3c7f237225a946ba737d
Author: vinodhini balusamy <me.vinob@gmail.com>
Date:   Tue Apr 24 08:54:48 2018 +1000

    "Fixed the unit tests"

commit 77432bbe01114bbd5be4d01881805f24b32b5883
Author: vinodhini balusamy <me.vinob@gmail.com>
Date:   Tue Apr 24 04:37:25 2018 +1000

    "Deleted unused imports"

commit 77c8dcc2d55aa724be8d0f3fc780e780de943d70
Author: vinodhini balusamy <me.vinob@gmail.com>
Date:   Tue Apr 24 02:25:35 2018 +1000

    "Additional code chamges to relocate common code"

commit 465be65dad559ab9e98b4b3e7ddc13c01dcd31f4
Author: vinodhini balusamy <me.vinob@gmail.com>
Date:   Mon Apr 23 23:54:55 2018 +1000

    "removed the whitespace charecters"

commit f5a9aa6c7c560d3eaa57e2ad25e877bccb963da4
Author: vinodhini balusamy <me.vinob@gmail.com>
Date:   Mon Apr 23 23:44:56 2018 +1000

    "updatseries and agentrelated function relocated"

## Please provide the following details to expedite Pull Request review:
## Description of change
This PR addresses code changes that refactor the common functions residing in update series folder to service folder as these functions are specific to service confirmed files and can be reused in other functionalities.

## QA steps
execute updateseries command by deploying ubuntu charm. Make sure the functionalities are working as before.

## Documentation changes
Does it affect current user workflow? CLI? API?
N/A
## Bug reference
Does this change fix a bug? Please add a link to it.
N/A
